source/rust_verify/src/verifier.rs

```diff
@@ -253,6 +253,7 @@ impl Verifier {
             (std::time::Duration::from_secs(2), report_fn)
         };
         let is_check_valid = matches!(**command, CommandX::CheckValid(_));
+        let is_singular = matches!(**command, CommandX::SingularCheckValid(..));
```

Comment:
How are recommends checks handled in singular? Are they ignored?

Reply:
I think the recommends check will simply run the same thing on Singular.